### PR TITLE
Fix get raster ee bug 

### PR DESCRIPTION
**Pull request by gena**
Fixing https://github.com/openearth/hydro-engine/issues/36

### CODE REVIEW

**Comment by gena** [comment]
Related to https://github.com/openearth/hydro-engine/issues/36

File: hydroengine_service/main.py [hunk 1]
```diff
@@ -594,7 +595,8 @@ def get_water_mask():
 
     water_mask_vector = get_water_mask_vector(region, scale, start, stop)
 
-    water_mask_vector = water_mask_vector.map(transform_feature(crs, scale))
+    if crs is not None and crs != 'EPSG:4326':
```
Comment (SiggyF) [inline hydroengine_service/main.py:598]:
```suggestion
    if crs and crs != 'EPSG:4326':
```

File: hydroengine_service/main.py [hunk 2]
```diff
@@ -634,7 +634,8 @@ def get_water_network():
     centerline = centerline.map(
         lambda line: line.set('length', line.length(scale / 10)))
 
-    centerline = centerline.map(transform_feature(crs, scale))
+    if crs is not None and crs != 'EPSG:4326':
```
Comment (SiggyF) [inline hydroengine_service/main.py:637]:
```suggestion
    if crs and crs != 'EPSG:4326':
```

File: hydroengine_service/main.py [hunk 3]
```diff
@@ -751,7 +752,8 @@ def add_elevation(pt):
     points = points.map(add_elevation)
     points = points.map(add_flow_accumulation)
 
-    points = points.map(transform_feature(crs, scale))
+    if crs is not None and crs != 'EPSG:4326':
```
Comment (SiggyF) [inline hydroengine_service/main.py:755]:
```suggestion
    if crs and crs != 'EPSG:4326':
```

**Review by SiggyF — APPROVED** [review]
Looks good.

**Pull request merged**